site_reliability_guardian_sample/manifest.yaml

```diff
@@ -0,0 +1,21 @@
+manifestVersion: 1.0
+projects:
+  - name: srg-config-as-code
+    path: srg-config-as-code
+
+environmentGroups:
+  - name: default
+    environments:
+      - name: DT_ENV_ID
+        url:
+          value: DT_ENV_URL
+        auth:
+          token:
+            name: API_TOKEN
+          oAuth:
+            clientId:
+              name: CLIENT_ID
+            clientSecret:
+              name: CLIENT_SECRET
+            tokenEndpoint:
```

Comment:
technically this property is not required (and not really documented) and is just need when testing internally/against environments that do not connect to the default SSO. 

Please remove the `tokenEndpoint` configuration to not confuse users :)